### PR TITLE
updating calentre's readme file

### PR DESCRIPTION
**Pull request by luisflores93**
updating calentre's readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,83 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+<img src = "https://github.com/fiizzy/calentre/blob/main/doc/images/logo.png?raw=true"  >
 
-## Getting Started
+# Calentre 📅💰
```
Comment (fiizzy) [inline README.md]: We need to change this to Calentre-Client

File: README.md [hunk 2]
```diff
@@ -1,36 +1,83 @@
...
 
-First, run the development server:
+Welcome to Calentre, the open-source alternative to Calendly! 🌟 Manage your appointments, get paid, and enjoy the scheduling experience.
```
Comment (fiizzy) [inline README.md]: "Welcome to the Client (invitee) side of Calentre. This repo contains the code that allows on invitee (client) to book a meeting based on the schedule of the admin (meeting owner) "

File: README.md [hunk 3]
```diff
@@ -1,36 +1,83 @@
...
 
-## Learn More
+<img src = "https://github.com/fiizzy/calentre/blob/main/doc/images/home_events.png?raw=true" >
 
-To learn more about Next.js, take a look at the following resources:
+---
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+<img src = "https://github.com/fiizzy/calentre/blob/main/doc/images/booking.png?raw=true" >
```
Comment (fiizzy) [inline README.md]: Let's take out this image, and replace it with Client image. A screenshot can be taken from your dev app.

File: README.md [hunk 4]
```diff
@@ -1,36 +1,83 @@
...
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+## Features ✨
 
-## Deploy on Vercel
+- **Scheduling:** Let clients book appointments with ease.
+- **Payment Integration:** Seamlessly get paid for your valuable time.
+- **Customization:** Tailor the scheduling process to match your style.
```
Comment (fiizzy) [inline README.md]: These features should be relative to Calentre-Client only.

File: README.md [hunk 5]
```diff
@@ -1,36 +1,83 @@
...
+
+Using Calentre is a breeze:
+
+1. Set the user `pmusa` in the url `http://localhost:3000/pmusa`.
+2. Select a meeting.
+3. Select a date and time for the meeting.
+4. Click on next.
+5. Under construction 🚧.
+
+For more, check the [User Guide](doc/user-guide.md).
```
Comment (fiizzy) [inline README.md]: Can you also edit this to also just talk about Calentre Client.

**Review by fiizzy — CHANGES_REQUESTED** [review]
Generally looking good. But we want the ReadMe to be focused only on the Client.

File: README.md (hunk 5 above)
Comment (luisflores93) [reply README.md]: @Jessica-Andrew could you help me with this content 😅

File: README.md (hunk 4 above)
Comment (luisflores93) [reply README.md]: @Jessica-Andrew could you help me with this content 😅

**Review by fiizzy — APPROVED** [review]
Going to approve this ✅

**Pull request merged**